transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -177,6 +180,15 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
+        HyperwalletInsight.getInstance().trackImpression(requireContext(), TAG,
+                getResources().getString(R.string.tag_group_transfer_method),
```

Comment:
mTransferMethodGroup = getString(R.string.tag_group_transfer_method);
initialise the variable commonly. or merge from PR 111
HyperwalletInsight.getInstance().trackImpression(requireContext(), TAG, mTransferMethodGroup,....);